src/extractDates.js

```diff
@@ -6,11 +6,18 @@ export default extractDates
 const parser = new chrono.Chrono
 parser.refiners.push(wholeMonth)
 
-function extractDates(str) {
+function extractDates(str, timezoneOffset) {
```

Comment:
Makes sense. Merge
On Thu, Jul 14, 2016 at 5:28 PM Chad Fawcett <notifications@github.com>
wrote:

> In src/extractDates.js
> <https://github.com/tickbin/filter-parser/pull/13#discussion_r70906236>:
>
> > @@ -6,11 +6,18 @@ export default extractDates
> >  const parser = new chrono.Chrono
> >  parser.refiners.push(wholeMonth)
> >
> > -function extractDates(str) {
> > +function extractDates(str, timezoneOffset) {
>
> I initially had a default. It would have needed to be the current timezone
> offset (moment().utcOffset()) otherwise it would always parse as UTC when
> no offset was provided. I thought it would be better to not have to provide
> one at all which means we don't have to change the other code relying on
> the parser.
>
> Then I changed it to the condition block as it was consistent with the
> tickbin-parser and I thought it was cleaner looking. V.S.
>
> function extractDates(str, timezoneOffset = moment().utcOffset()) {
>   ...
> }
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tickbin/filter-parser/pull/13/files/bc791e0b81965938e4a1d77a3a5528e90995f7ef#r70906236>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVoL2O6b_RR3dd6C6pwj0V1ZctZtD0ks5qVtRFgaJpZM4JM8oK>
> .
>